sdk/digitaltwins/Azure.DigitalTwins.Core/tests/TestSettings.cs

```diff
@@ -19,13 +19,6 @@ public class TestSettings
 
         // If these environment variables exist in the environment, their values will replace (supersede) config.json values.
 
-        private static readonly string s_adtInstanceEndpointUrlEnv = $"{AdtEnvironmentVariablesPrefix}_ADT_INSTANCE_ENDPOINT_URL";
-        private static readonly string s_applicationTenantIdEnv = $"{AdtEnvironmentVariablesPrefix}_TENANT_ID";
-        private static readonly string s_applicationClientIdEnv = $"{AdtEnvironmentVariablesPrefix}_CLIENT_ID";
-        private static readonly string s_applicationClientSecretEnv = $"{AdtEnvironmentVariablesPrefix}_CLIENT_SECRET";
-        private static readonly string s_digitalTwinHostnameEnv = $"{AdtEnvironmentVariablesPrefix}_URL";
-        private static readonly string s_digitalTwinTestMode = $"AZURE_IOT_TEST_MODE";
-
         public static TestSettings Instance { get; private set; }
 
         public RecordedTestMode TestMode { get; set; }
```

Comment:
I have not been able to move this out of TestSettings yet. So the way to run tests in different modes is still the same.